coral/src/app/features/dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,187 @@
+import { Box, Card, Template } from "@aivenio/aquarium";
+import zip from "lodash/zip";
+import StatsDisplay from "src/app/components/StatsDisplay";
+import { AreaChart, Axis, BarChart, Tooltip } from "@aivenio/aquarium/charts";
+
+const mockData = {
+  topicsPerTeamPerEnvOverview: {
+    data: [128, 32, 1],
+    labels: ["DEV", "TST", "PRD"],
+    colors: ["Green", "Green", "Green"],
+    options: {
+      title: {
+        display: true,
+        text: "Topics per cluster (Ospo) (Total 161)",
+        position: "bottom",
+        fontColor: "red",
+      },
+    },
+    titleForReport: "Topics per cluster (Ospo)",
+    xaxisLabel: "Clusters",
+    yaxisLabel: "Topics",
+  },
+  activityLogOverview: {
+    data: [
+      3, 2, 1, 2, 1, 1, 5, 3, 2, 1, 7, 2, 7, 32, 1, 6, 7, 2, 1, 3, 3, 4, 3, 2,
+      2, 3, 10, 7, 3, 1,
+    ],
+    labels: [
+      "17-Jun-2023",
+      "20-Jun-2023",
+      "21-Jun-2023",
+      "23-Jun-2023",
+      "26-Jun-2023",
+      "27-Jun-2023",
+      "28-Jun-2023",
+      "29-Jun-2023",
+      "30-Jun-2023",
+      "03-Jul-2023",
+      "04-Jul-2023",
+      "05-Jul-2023",
+      "06-Jul-2023",
+      "07-Jul-2023",
+      "10-Jul-2023",
+      "11-Jul-2023",
+      "12-Jul-2023",
+      "13-Jul-2023",
+      "14-Jul-2023",
+      "17-Jul-2023",
+      "19-Jul-2023",
+      "20-Jul-2023",
+      "21-Jul-2023",
+      "24-Jul-2023",
+      "25-Jul-2023",
+      "26-Jul-2023",
+      "27-Jul-2023",
+      "28-Jul-2023",
+      "31-Jul-2023",
+      "02-Aug-2023",
+    ],
+    colors: [
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+      "Green",
+    ],
+    options: {
+      title: {
+        display: true,
+        text: "Requests per day (Ospo) (Total 127)",
+        position: "bottom",
+        fontColor: "red",
+      },
+    },
+    titleForReport: "Requests per day (Ospo)",
+    xaxisLabel: "Days",
+    yaxisLabel: "Activities",
+  },
+};
+
+const Dashboard = () => {
+  const requestsData = zip(
```

Comment:
ha! Never used `zip`, pretty neat!

Reply:
It's even better with `zipWith`, this is just a quick and dirty way to get some data in there haha